roles/netapp-trident/tasks/main.yml

```diff
@@ -166,26 +36,30 @@
 - name: Create Trident backend(s)
   command: /root/trident-installer/tridentctl -n trident create backend -f /root/trident-installer/backend-{{ item.backendName }}.json
   with_items: "{{ backends_to_create }}"
+  retries: 10
+  delay: 30
+  register: result
+  until: result.rc == 0
   when: create_backends == true
   run_once: true
 
-- name: Create NetApp StorageClass(es) (Ubuntu)
-  k8s:
-    state: present
-    definition: "{{ item }}"
+- name: Copy StorageClass config file(s) to control node
+  copy:
+    content: "{{ item }}"
+    dest: /root/trident-installer/sc-{{ item.metadata.name }}.yaml
   with_items: "{{ storageClasses_to_create }}"
+  when: create_StorageClasses == true
   run_once: true
-  environment:
-    PYTHONHOME: '{{ deepops_venv }}'
-  when: create_StorageClasses == true and ansible_distribution == 'Ubuntu'
 
-- name: Create NetApp StorageClass(es) (Red Hat / CentOS)
-  k8s:
+- name: Create StorageClasses
+  kube:
```

Comment:
Does this work with a standard/default DeepOps install? I seem to recall seeing a bunch of TODOs around this and most of are kubectl commands directly use the binary throuh "shell".

I'll take your word for it if you've tested this.

Reply:
@supertetelman - I actually switched to using the "kube" module instead of the "k8s" module because I was running into a bunch of weird Python interpreter when using the "k8s" module with Ubuntu 20.04. I noticed that Kubespray was using the "kube" module instead, so I modeled my steps after theirs. I tested all of this on a clean install using standard/default DeepOps, and it worked.